src/examples/scala/cp.scala

```diff
@@ -0,0 +1,13 @@
+object cp extends loamstream.LoamFile {
+  val fileIn = store("fileIn.txt").asInput
+  val fileTmp1 = store
+  val fileTmp2 = store
+  val fileOut1 = store("fileOut1.txt")
+  val fileOut2 = store("fileOut2.txt")
+  val fileOut3 = store("fileOut3.txt")
```

Comment:
Again, comments explaining why I should use `store` vs. just the filename? Also, this would be good to clarify:

```scala
val fileOut1 = store("a.txt")
val fileOut2 = store("a.txt")

fileOut1 == fileOut2 // true?
```